src/sdk.jsx

```diff
@@ -128,6 +130,11 @@ class OpenForm {
     ).href;
   }
 
+  getInitialDataReference() {
```

Comment:
```suggestion
  extractInitialDataReference() {
```
if I see a method `foo.getInThereLewis()`, I expect to get the thing when I call, not a side-effect that sets it on the instance.

So I'd go for `extract` or `store` or something like that

Reply:
This method was written for Lewis in a Ferrari; you expect to get something but actually end up with nothing